academy/3-cosmos-hub/1-cosmos-hub-overview

```diff
@@ -0,0 +1,27 @@
+---
```

Comment:
Potentially change naming of file if to be included as landing page for separate chapter